cli/internal/ui/ui.go

```diff
@@ -63,13 +54,38 @@ func Rainbow(text string) string {
 	return strings.Join(rainbowStr, "")
 }
 
+type stripAnsiWriter struct {
+	wrappedWriter io.Writer
+}
+
+func (into *stripAnsiWriter) Write(src []byte) (n int, err error) {
+	into.wrappedWriter.Write(ansiRegex.ReplaceAll(src, []byte{}))
```

Comment:
This looks like it is losing the return values?

I think in general we prefer not to use named return variables for this reason and general readability.